compatibility_server/compatibility_checker_server.py

```diff
@@ -165,9 +165,16 @@ def log_message(self, format, *args):
         type=int,
         default=8888,
         help='port to which the server should bind')
+    parser.add_argument(
+        '--export_metrics',
+        action='store_true',
+        help='whether or not metrics should be exported to stackdriver')
     args = parser.parse_args()
     logging.info('Running server with:\n%s', pprint.pformat(vars(args)))
 
+    global EXPORT_METRICS
```

Comment:
Prefix this with an underscore and move it before any functions:
http://google.github.io/styleguide/pyguide.html#254-decision

You could also add `check` to a class so the constructor can take this as an argument. I'd probably do that.

Reply:
addressed this by enabling exporters on the spot